cmd/svcat/binding/unbind_cmd.go

```diff
@@ -35,6 +41,9 @@ type unbindCmd struct {
 
 	instanceName string
 	bindingNames []string
+	abandon      bool
+	stdin        io.Reader
```

Comment:
I don't understand why this is defined as a field on the command struct. Does COBRA do something special to populate this for you? where is it getting instantiated?